devices/HumanDataCollector/HumanDataCollector.cpp

```diff
@@ -98,6 +238,18 @@ class HumanDataCollector::impl
 
     yarp::os::BufferedPort<yarp::os::Bottle> dynamicsJointNamesDataPort;
     yarp::os::BufferedPort<yarp::sig::Vector> jointTorquesDataPort;
+
+
+    // MATIO Logging
+    bool matioLogger = false;
+    const std::experimental::filesystem::path originalWorkingDirectory = std::experimental::filesystem::current_path();
+    std::string matLogDirectory = "";
+    std::string matLogFileName = "matLogFile.mat"; //TODO: Improve file handling for multiple file logging
```

Comment:
the name of the file might be also passed as a configuration

Reply:
Done https://github.com/robotology/human-dynamics-estimation/pull/181/commits/b8de3bf592fbf86f3100ea63ddee03fddd0eb34d